media-libs/freetype/freetype-2.6.2.recipe

```diff
@@ -62,6 +66,7 @@ BUILD()
 INSTALL()
 {
 	make install
+	strip$secondaryArchSuffix -s $libDir/libfreetype.so.6.12.2
```

Comment:
It makes sense to use the strip version that matches the compiler. I'm not sure it makes a real difference, but we are sure they parse the ELF files in compatible ways.